potato/flask_server.py

```diff
@@ -45,6 +46,9 @@
     go_to_id,
 )
 import state
+from scripts.init_user_db import init_user_db
+
+POTATO_HOME = os.environ.get("POTATO_HOME")

```

Comment:
I'm suggesting here that POTATO requires an environment variable `POTATO_HOME` to be set that points to the root directory of the repo -- my humble opinion is that this will make package management / path imports a lot easier, but happy to discuss.